metadata/src/test/java/org/apache/kafka/image/MetadataVersionChangeTest.java

```diff
@@ -30,31 +30,31 @@
 @Timeout(value = 40)
 public class MetadataVersionChangeTest {
 
-    private static final MetadataVersionChange CHANGE_3_0_IV1_TO_3_3_IV0 =
-        new MetadataVersionChange(IBP_3_0_IV1, IBP_3_3_IV0);
+    private static final MetadataVersionChange CHANGE_3_3_IV3_TO_3_6_IV0 =
```

Comment:
We just need two different MVs here. Could we use MetadataVersion.MINIMUM_VERSION and MetadataVersion.latestProduction?